packages/lib/src/index.ts

```diff
@@ -179,5 +163,20 @@ export async function waitForWindowAI(opts = DEFAULT_WAIT_OPTIONS) {
 export const getWindowAI = async (opts = DEFAULT_WAIT_OPTIONS) => {
   // wait until the window.ai object is available
   await waitForWindowAI(opts)
+
+  // Context: https://github.com/alexanderatallah/window.ai/pull/43#discussion_r1178316153
+  const _getCompletion = globalThis.window.ai.getCompletion
+  globalThis.window.ai.getCompletion = async (input, options = {}) => {
```

Comment:
Would this be cleaner if we just call `await _getCompletion`, check the type of the return result, and if it's an Array, then return it, otherwise wrap it in an array?

Reply:
That makes sense lol - I was blindsided by focusing on the options object as the source of this behavior and didn't think about just checking output itself.